src/frontend/components/Chat/SharedChatComponents/InputDisplayNameOverlay/index.tsx

```diff
@@ -0,0 +1,175 @@
+
+import { InputBar } from 'components/Chat/SharedChatComponents/InputBar';
+import { ExitCrossSVG } from 'components/SVGIcons/ExitCrossSVG';
+import { QuestionMarkSVG } from 'components/SVGIcons/QuestionMarkSVG';
+import { converse } from 'utils/window';
+import { InputDisplayNameIncorrectAlert } from './InputDisplayNameIncorrectAlert/index';
+
+const messages = defineMessages({
+  closeButtonTitle: {
+    defaultMessage: 'Click this button to close the overlay.',
+    description: 'A title describing the close button action',
+    id: 'components.InputDiplayNameOverlay.closeButtonTitle',
+  },
+  inputAnonymousKeywordForbiddenAlertMessage: {
+    defaultMessage: 'Keyword "anonymous" is not allowed.',
+    description:
+      'An alert message explaining why the entered display name is invalid.',
+    id: 'components.InputDisplayNameOverlay.inputAnonymousKeywordForbiddenAlertMessage',
+  },
+  inputDisplayNameInformative: {
+    defaultMessage:
+      "The display name is the pseudonym you will be authentificated with on this live. You won't be able to change during the live. Other participants will see you with this name. Instructor will however be able to see your geniune identity if you have previously identified yourself with the LTI.",
+    description:
+      'An informative text about the display name which is asked to enter.',
+    id: 'components.InputDisplayNameOverlay.inputDisplayNameInformative',
+  },
+  inputDisplayNameLabel: {
+    defaultMessage: 'Display name',
+    description: 'An label describing the input below.',
+    id: 'components.InputDisplayNameOverlay.inputDisplayNameLabel',
+  },
+  inputDisplayNamePlaceholder: {
+    defaultMessage: 'Enter your desired display name...',
+    description: 'The input bar to fill your display name.',
+    id: 'components.InputBar.inputDisplayNamePlaceholder',
+  },
+  inputTooShortAlertMessage: {
+    defaultMessage: 'Min lenght is 3 characters.',
+    description:
+      'An alert message explaining why the entered display name is invalid.',
+    id: 'components.InputDisplayNameOverlay.inputTooShortAlertMessage',
+  },
+  inputTooLongAlertMessage: {
+    defaultMessage: 'Max lenght is 15 characters.',
+    description:
+      'An alert message explaining why the entered display name is invalid.',
+    id: 'components.InputDisplayNameOverlay.inputTooLongAlertMessage',
+  },
+});
+
+interface InputDisplayNameOverlayProps {
+  setInputBarActive: React.Dispatch<React.SetStateAction<boolean>>;
+  setOverlay: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+export const InputDisplayNameOverlay = ({
+  setInputBarActive,
+  setOverlay,
+}: InputDisplayNameOverlayProps) => {
+  const intl = useIntl();
+  const [alertsState, setAlertsState] = useState<string[]>([]);
+
+  const processDisplayName = (displayName: string): number => {
+    const alerts: string[] = [];
+    if (
+      displayName.includes('Anonymous') ||
+      displayName.includes('anonymous')
```

Comment:
testing it directly tolowercase ?wdyt

Reply:
changed in commit [d3c2a0](https://github.com/openfun/marsha/pull/1276/commits/d3c2a037d1f2b70d6fd332d3c8951c3bea0df3a5)